packages/@coorpacademy-player-services/src/types.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+import type {Action, Content} from '@coorpacademy/progression-engine';
+
+type Url = string;
+
+type JwpOptions = {
+  customProps: {
+    aspectratio: '16:9' | '4:3',
```

Comment:
IMO this should be in another type like `JwpAspectRatio`.
Btw, what is Jwp ?

Reply:
renamed `JwPlayerOptions`, ✅